src/cs50/sql.py

```diff
@@ -357,6 +365,18 @@ def shutdown_session(exception=None):
                 elif command in ["DELETE", "UPDATE"]:
                     ret = result.rowcount
 
+                # If COMMIT or ROLLBACK, turn on autocommit mode
+                elif command in ["COMMIT", "ROLLBACK"] and "TO" not in statement:
+                    session.close()
+                    self._session = None
+
+
```

Comment:
One too many \n?

Reply:
Included the extra one to denote a change in control flow (next statement is a new `if` separate from the previous ones). Can change if you prefer.

Reply:
Just single blanks within functions! Two between functions, per PEP.

Reply:
Sounds good.